client/src/components/ManageBooking/Calendar/calendar.css

```diff
@@ -0,0 +1,148 @@
+.react-calendar {
```

Comment:
Is this file auto-generated, or is it necessary for `react-calendar` to work? You can ignore this comment if it is.

Reply:
yes Ugur, it's styling for react-calendar package and I have to keep it